src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ConfiguredExceptionsStoreTests.cs

```diff
@@ -0,0 +1,259 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using Microsoft.Diagnostics.Monitoring.WebApi.Exceptions;
+using Microsoft.Diagnostics.Tools.Monitor.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    public sealed class ConfiguredExceptionsStoreTests
+    {
+        private const ulong DefaultExceptionGroupId = 1;
+
+        /// <summary>
+        /// Validates that a <see cref="ConfiguredExceptionsStore"/> can be instantiated.
+        /// </summary>
+        [Fact]
+        public async Task ConfiguredExceptionsStore_Creation()
+        {
+            await using ConfiguredExceptionsStore store = new ConfiguredExceptionsStore(7);
+        }
+
+        /// <summary>
+        /// Validates that a <see cref="ConfiguredExceptionsStore"/> will throw with invalid top level limit.
+        /// </summary>
+        [Fact]
+        public void ConfiguredExceptionsStore_LimitZero_ThrowsException()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(() => new ConfiguredExceptionsStore(0));
+        }
+
+        /// <summary>
+        /// Validates adding a single exception to the store with a limit of one will hold the exception instance.
+        /// </summary>
+        [Fact]
+        public async Task ConfiguredExceptionsStore_LimitOne_OnlyExceptionRemains()
+        {
+            int ExceptedCount = 1;
```

Comment:
I'm _assuming_ this is supposed to be `ExpectedCount` - if so it looks like this copy-paste propagated to a bunch of these test methods